ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -145,11 +149,41 @@ private fun ColumnScope.Template2MainContent(
             Spacer(Modifier.weight(1f))
         }
 
+        AnimatedPackages(state, packageSelectionVisible, viewModel, childModifier)
+
+        if (state.isInFullScreenMode) {
+            Spacer(Modifier.weight(1f))
+        }
+    }
+}
+
+@Composable
+private fun AnimatedPackages(
+    state: PaywallState.Loaded,
+    packageSelectionVisible: Boolean,
+    viewModel: PaywallViewModel,
+    childModifier: Modifier,
+) {
+    val packagesContentAlignment = if (state.isInFullScreenMode) {
+        Alignment.TopStart
+    } else {
+        Alignment.BottomCenter
+    }
+    Box(contentAlignment = packagesContentAlignment) {
+        AnimatedVisibility(
+            visible = !packageSelectionVisible,
+            enter = fadeIn(animationSpec = tween(delayMillis = 200)),
+            exit = fadeOut(animationSpec = tween(delayMillis = 200)),
```

Comment:
This is `UIConstant.defaultAnimationDurationMillis`

Reply:
Also might want to use `UIConstant.defaultAnimation`?

Reply:
Oh, how come there's a delay?

Reply:
Without it, the fade animation was masked by the movement animation of the packages. By delaying it, you can slightly see the fade at the end when closing the package selector. Tried to mimick the feel from iOS.